Constitution.txt

```diff
@@ -1,490 +1,264 @@
-We the People of the United States, in Order to form a more perfect Union, establish Justice, insure domestic Tranquility, provide for the common defence, promote the general Welfare, and secure the Blessings of Liberty to ourselves and our Posterity, do ordain and establish this Constitution for the United States of America.
+We the People of these United States of America, in Order to form a more stable Union, establish Justice, ensure domestic Tranquility, provide for the common defense, and secure the Blessings of Liberty to ourselves and our Posterity, do ordain and re-establish this Constitution for these United States of America.
 
 Article I (Article 1 - Legislative)
 Section 1
-All legislative Powers herein granted shall be vested in a Congress of the United States, which shall consist of a Senate and House of Representatives.
+All legislative Powers herein granted shall be vested in a Congress of these United States, which shall consist of a Senate and House of Representatives. Only such powers as are specifically enumerated shall be so vested, and all other legislative matters shall remain with the People of the several States, or delegated to the Governments of the several States by the People thereof.
 
 Section 2
-1: The House of Representatives shall be composed of Members chosen every second Year by the People of the several States, and the Electors in each State shall have the Qualifications requisite for Electors of the most numerous Branch of the State Legislature.
+1: The House of Representatives shall be composed of Members chosen every second Year by the Citizens of the several States, and the Electors in each State shall have the Qualifications requisite for Electors of the most numerous Branch of the State Legislature.
```

Comment:
This is the first instance where I begin to draw a clear delineation between the People and the Citizens. It is made more explicit later in the document.